django_bestiary/projects/views.py

```diff
@@ -105,3 +107,30 @@ def import_from_file(request):
         print("Repositories loaded", nrepos)
 
     return index(request)
+
+
+def export_to_file(request):
+
+    if not os.path.exists('.exported'):
+        os.mkdir(".exported")
+
+    if request.method == "POST":
+        ecosystem = request.POST["ecosystem"]
+        file_name = "projects_%s.json" % ecosystem
+        file_path = os.path.abspath(os.path.curdir) + "/.exported/" + file_name
+
+        task_init = time()
+        (nprojects, nrepos) = export_projects(file_path, ecosystem)
```

Comment:
Probably we can avoid writing the data to a file and reading from it, and just use the memory directly. But for this iteration is it ok using this approach.